streams/ar.m3u

```diff
@@ -154,21 +154,161 @@ https://5f700d5b2c46f.streamlock.net/vertv/vertv/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal5Tucuman.ar" status="online",Canal 5 (Tucumán) [Not 24/7]
+https://videohd.live:19360/8090/8090.m3u8
+#EXTINF:-1 tvg-id="Canal6EntreRios.ar" status="online",Canal 6 (Entre Ríos)
+https://stmvideo1.livecastv.com/canal6er/canal6er/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal6MardePlata.ar" status="online",Canal 6 (Mar de Plata)
+https://59537faa0729a.streamlock.net/canal6mdp/canal6mdp/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal6Moreno.ar" status="online",Canal 6 (Moreno)
+https://stream.arcast.com.ar/canal6moreno/canal6moreno/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal6Posadas.ar" status="online",Canal 6 (Posadas)
+https://iptv.ixfo.com.ar:30443/live/c6digital/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal79MardelPlata.ar" status="online",Canal 79 (Mar del Plata)
+https://streamconex.com:1936/mardelplata/mardelplata/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal79SanJuan.ar" status="online",Canal 79 (San Juan)
+https://streamconex.com:1936/sanjuan/sanjuan/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal79SantaClaradelMar.ar" status="online",Canal 79 (Santa Clara del Mar)
+https://streamconex.com:1936/santaclara/santaclara/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal7Jujuy.ar" status="online",Canal 7 (Jujuy)
+https://stream.arcast.live/canal7jujuy/ngrp:canal7jujuy_all/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal2Sanagasta.ar" status="online",Canal 2 (Sanagasta)
+https://stmvideo1.livecastv.com/canal2/canal2/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal4SanJuan.ar" status="online",Canal 4 (San Juan)
+http://streamlov.alsolnet.com/canal4sanjuan/live/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal907FM.ar" status="online",Canal 907 FM
+https://panel.dattalive.com/canal907/canal907/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal9Litoral.ar" status="online",Canal 9 Litoral
+https://stream.arcast.live/ahora/ahora/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal9RioCuarto.ar" status="online",Canal 9 Río Cuarto
+https://streamcasthd.com:1936/canal9riocuartotv/canal9riocuartotv/playlist.m3u8
+#EXTINF:-1 tvg-id="EPSTV.ar" status="online",EPS TV
+https://ott.argentinatv.live/vivo/noticiaseps/playlist.m3u8
+#EXTINF:-1 tvg-id="LTOTelevision.ar" status="online",LTO Televisión [Not 24/7]
+https://ltocatamarca.github.io/ltotv.m3u8
+#EXTINF:-1 tvg-id="CosmosTV.ar" status="online",Cosmos TV
+https://tv.mediacp.eu:19360/cosmos/cosmos.m3u8
+#EXTINF:-1 tvg-id="CPEtv.ar" status="online",CPEtv
+https://stream.arcast.live/cpe/ngrp:cpe_all/playlist.m3u8
+#EXTINF:-1 tvg-id="ComarcaTV.ar" status="online",Comarca TV
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.twitch.tv/canalccordoba
+#EXTINF:-1 tvg-id="CL3Cablevision.ar" status="online",CL3 Cablevision
+https://videostream.shockmedia.com.ar:1936/cl3cable/cl3cable/playlist.m3u8
+#EXTINF:-1 tvg-id="ChilecitoTV.ar" status="online",Chilecito TV
+https://vivo.solumedia.com:19360/grupoemail/grupoemail.m3u8
+#EXTINF:-1 tvg-id="CatamarcaTV.ar" status="error",Catamarca TV (288p)
+https://5f700d5b2c46f.streamlock.net/catamarcatelevision/catamarcatelevision/playlist.m3u8
```

Comment:
Supposing Catamarca TV is not 24/7, since both links are down.

Reply:
I guess. But the old link is possibly outdated, would be necessary check it again in the future.